format/src/main/thrift/schema.thrift

```diff
@@ -203,9 +203,9 @@ struct ParentColumnTableRelation {
    3: required string columnName
 }
 
-struct DataMapSchema  {
+struct IndexSchema  {
```

Comment:
Changing thrift member name will leads to compatibility issue. Old store will have as object of DataMapSchema, type cast may fail. Have tried reading old store that has datamap schema ?

Reply:
@niuge01 , @QiangCai : please check in previous merged PR also for compatibility issues. better to test old store once.

Reply:
These changes added in `schema.thrift` are to support pre-aggregate. I will remove those changes from `schema.thrift` file